Source/Modules/ModuleInput.cpp

```diff
@@ -366,3 +366,16 @@ bool ModuleInput::CleanUp()
 
 	return true;
 }
+
+void ModuleInput::Rumble(Uint16 strengthLeft, Uint16 strengthRight, int duration) const
+{
+	SDL_GameController* controller = FindController();
+
+	if (controller != nullptr)
+	{
+		if (SDL_GameControllerRumble(controller, strengthLeft, strengthRight, duration) != 0)
+		{
+			LOG_ERROR("Error on controller rumble");
```

Comment:
Could we get the error? I think we can use [`SDL_GetError`](https://wiki.libsdl.org/SDL2/SDL_GetError)
```suggestion
			LOG_ERROR("Error on controller rumble: {}", SDL_GetError());
```

Reply:
I didn't know we could do this. Nice one!